bin/test

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# default variables
+export DEVELOPMENT=1
+export DJANGO_CONFIGURATION=Test
```

Comment:
We can get rid of all the DJANGO_CONFIGURATION variables, those are specific to atmo